compiler/rustc_infer/src/infer/mod.rs

```diff
@@ -670,6 +670,22 @@ pub struct CombinedSnapshot<'a, 'tcx> {
 }
 
 impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
+    /// calls `tcx.try_unify_abstract_consts` after
+    /// canonicalizing the consts
+    pub fn canonical_try_unify_abstract_consts(
```

Comment:
```suggestion
    pub fn try_unify_abstract_consts(
```
we already have the same pattern for `const_eval_resolve`